docs/site-replication/run-multi-site-ldap.sh

```diff
@@ -181,6 +185,14 @@ if [ $? -ne 0 ]; then
     exit_1;
 fi
 
+sleep 10
+./mc stat minio3/newbucket/lrgfile
```

Comment:
we should also do `mc cat` and check the checksum before and after `mc cat ./lrgfile | md5sum` and then `mc cat minio3/newbucket/lrgfile | md5sum` must match. 

It is better than `mc stat` IMO